auth-web/src/components/auth/common/ExpandableHelp.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div>
+    <div
+      class="help-btn top mb-8"
+      @click="helpToggle = !helpToggle"
+    >
+      <v-icon color="primary">
+        mdi-help-circle-outline
+      </v-icon>
+      <span
+        v-if="!helpToggle"
+        class="pl-2"
+      >Help with Starting and Managing a Business</span>
+      <span
+        v-else
+        class="pl-2"
+      >Hide Help</span>
+    </div>
+    <v-expand-transition>
+      <div
+        v-show="helpToggle"
+        class="help-section pa-4 my-3 mb-8"
+      >
+        <slot name="content" />
+        <div
+          class="help-btn bottom text-right d-flex"
+          @click="helpToggle = !helpToggle"
+        >
+          Hide Help
+        </div>
+      </div>
+    </v-expand-transition>
+  </div>
+</template>
+
+<script lang="ts">
+import { ref } from '@vue/composition-api'
+
+export default {
+  setup () {
+    const helpToggle = ref(false)
+    return { helpToggle }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+      .help-btn {
+        cursor: pointer;
+        color: $app-blue;
+        vertical-align: middle;
```

Comment:
You can use a Vuetify class in the template for this property.

Reply:
I was thinking about ["align-self-center"](https://v2.vuetifyjs.com/en/styles/flex/#flex-align-self) but I guess, in the end, you didn't need it.

Reply:
Do not need it. Looks pretty close to UX pin after a lot of CSS changes